.travis.yml

```diff
@@ -1,11 +1,6 @@
 language: java
 dist: trusty
 sudo: false
-addons:
-  apt:
-    packages:
-      - oracle-java8-installer
-      - oracle-java9-installer
```

Comment:
I believe these are not required anymore. Java 8 and 9 are just stable, included JDKs.

Reply:
👍